pkg/hardwareutils/bmc/access_test.go

```diff
@@ -679,6 +682,13 @@ func TestStaticDriverInfo(t *testing.T) {
 				t.Fatalf("Unexpected bios interface %q, expected %q",
 					acc.BIOSInterface(), tc.bios)
 			}
+			if acc.RAIDInterface() != tc.raid {
```

Comment:
Does this really achieve anything other than making you change the code in two places whenever you change a driver?

Reply:
I don't know, it's an established pattern here. Better visibility at least?

Reply:
An established pattern of busywork ;)

(Checking stuff that is inferred from the URL, like the driver name and the boot interface, makes sense and is the purpose of this test. The rest is pure cargo-cult.)